src/api/objects/InMemorySearchWorker.js

```diff
@@ -26,16 +26,27 @@
 (function () {
     // An object composed of domain object IDs and models
     // {id: domainObject's ID, name: domainObject's name}
-    const indexedItems = {};
+    const indexedDomainObjects = {};
```

Comment:
this is all basically a copy of what was done above

Reply:
yeah, there's a lot of code duplication here unfortunately as the worker code need to be in a separate & contained file. once safari has shared workers, we can remove a lot of the "local" implementation code